### PR TITLE
Feature: Add "toggle coordinates" button in Puzzles

### LINKED ISSUE #1454: Puzzles should have "toggle coordinates" option
**Is your feature request related to a problem? Please describe.**
As shown below, many puzzles refer to coordinates, and coordinates even get automatically adjusted for the different board transformations, but if you have coordinates off, it's hard to understand them. You have to go find some other game, click "toggle coordinates" there, then reload the puzzle page.

**Describe the solution you'd like**
There should be a "toggle coordinates" button somewhere on the puzzle page.

**Describe alternatives you've considered**
Linking coordinates with some kind of highlight feature like we do in the game chat could be just as helpful, but I'm assuming that's harder.

**Additional context**
image

### PR DESCRIPTION
**Pull request by melissa-z**
Fixes #1454
## Proposed Changes

Hi, first time contributing. Love OGS! 

  - Add a "toggle coordinates" button in Puzzles
  - Add `title` attribute to buttons
  - Minor changes to clean up button styling 
     - Remove border outline and add margin instead (feels more modern and the spacing improves accessibility)
     - Fix active state border color

Screen Recording 2021-04-18 at 2 19 52 PM

Screen Shot 2021-04-18 at 2 20 43 PM

### CODE REVIEW

**Comment by benjaminpjones** [comment]
Awesome! One question: did you consider using the existing coordinates icon for the button?  To me it feels like the full "Toggle Coordinates" takes up a lot of space compared to the other buttons.

image

**Comment by anoek** [comment]
https://github.com/online-go/online-go.com/blob/devel/src/views/Game/Game.tsx#L3143 shows the usage, you should just be able to drop in `<i className="ogs-coordinates" />` for that icon I think

**Comment by melissa-z** [comment]
That's a great point, will do!

**Comment by melissa-z** [comment]
I switched out "Toggle Coordinates" for the icon, and also added some `title` attributes to the icon-only buttons so that a text description will appear on hover. Any thoughts?

**Comment by benjaminpjones** [comment]
Looks nice! Thanks!  And yeah I think the tooltips are helpful.

File: src/views/Puzzle/Puzzle.tsx [hunk 1]
```diff
@@ -779,6 +779,7 @@ export class Puzzle extends React.Component<PuzzleProperties, PuzzleState> {
             <div className="btn-container">
                 <div className="btn-group">
                     <button type="button"
+                        title="Flip Diagonal"
```
Comment (benjaminpjones) [inline src/views/Puzzle/Puzzle.tsx:782]: What I meant in the other comment:

```title={_("Flip diagonal")}```

Alternatively, if you want to add some context:

```title={pgettext("Puzzle options", "Flip Diagonal")```

The translators will be able to see this in their dashboard and add a translation later.

Comment (melissa-z) [reply src/views/Puzzle/Puzzle.tsx]: Fixed!

Comment (benjaminpjones) [reply src/views/Puzzle/Puzzle.tsx]: Cool, thanks!

**Comment by anoek** [comment]
Thanks for the improvements!

**Pull request merged**